basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/PwdStep/help.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    Returns the current directory as a string.
+<p><pre>
+def x = pwd()
```

Comment:
Not really. I wanted to show that the step itself does not perform anything internally, it only returns a value, but this is already clear from the sentence before, so... removing.